common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -963,6 +962,13 @@ div.problem .action {
   align-items: center;
   flex-wrap: wrap;
 
+  .problem-action-buttons-wrapper {
```

Comment:
Not a huge fan of this name but it was already being used and I didn't want to mess with it